src/main/java/fact/utils/PreviousEventInfo.java

```diff
@@ -18,7 +18,11 @@
     @Parameter(required = true)
     public String outputKey = null;
 
-    int limitEvents = 20;
+    @Parameter(required = false, description = "The key containing the UnixTimeUTC")
```

Comment:
Why is this a parameter?

Reply:
For the case someone wants to set it, we have an variable in 'settings.properties' for this but no way to set it in this processor (only in the jumpRemoval). So either this one goes or, the thing in setting.properties makes little sense because it is a constant.